draft-yasskin-http-origin-signed-responses.md

```diff
@@ -386,6 +386,52 @@ extended diagnostic notation from {{?I-D.ietf-cbor-cddl}} appendix G:
 ]
 ~~~
 
+## Loading a certificate chain ## {#cert-chain-format}
+
+The resource at a signature's `certUrl` MUST have the
+`application/cert-chain+cbor` content type, MUST be canonically-encoded CBOR
+({{canonical-cbor}}), and MUST match the following CDDL:
+
+~~~cddl
+cert-chain = [
+  "📜⛓", ; U+1F4DC U+26D3
+  + {
+    cert: bytes,
+    ? ocsp: bytes,
+    ? sct: bytes,
+    * tstr => any,
+  }
+]
+~~~
+
+The first item in the CBOR array is treated as the end-entity certificate, and
+the client will attempt to build a path ({{?RFC5280}}) to it from a trusted root
+using the other certificates in the chain.
+
+1. Each `cert` value MUST be a DER-encoded X.509v3 certificate ({{!RFC5280}}).
+   Other key/value pairs in the same array item define properties of this
+   certificate.
+1. The first certificate's `ocsp` value if any MUST be a complete, DER-encoded
+   OCSP response for that certificate (using the ASN.1 type `OCSPResponse`
+   defined in {{!RFC2560}}). Subsequent certificates MUST NOT have an `ocsp`
+   value.
```

Comment:
Re "subsequent certificates", I suspect this may end up being a bit controversial since the TLSWG intentionally went the other way with TLS 1.3 and made it easy to encode OCSP staples on intermediates. But I'll leave that for @sleevi and others to wrestle out later. :-)

(In our TLS 1.3 implementation, we tolerate metadata on intermediates but ignore them for now. We'll start processing them if we have something useful to do with them.)